specification/README.md

```diff
@@ -10,6 +10,7 @@ Semantic Conventions are defined for the following areas:
 * [Exceptions](exceptions/README.md): Semantic Conventions for Exceptions.
 * [FaaS](faas/README.md): Semantic Conventions for Function as a Service (FaaS) operations.
 * [CloudEvents](cloudevents/README.md): Semantic Conventions for the CloudEvents specification.
+* [Feature Flags](http/README.md): Semantic Conventions for feature flag evaluations.
```

Comment:
Wrong link

Reply:
Thanks for spotting this! 
Fixing it in https://github.com/open-telemetry/semantic-conventions/pull/158.